docs/modules/ROOT/pages/how-tos/exoscale/install.adoc

```diff
@@ -248,25 +207,16 @@ export HIERADATA_REPO_TOKEN=$(echo "${HIERADATA_REPO_SECRET}" | jq -r '.token')
 
 === OpenShift Installer Setup
 
-For the following steps, change into a clean directory (for example a directory in your home).
-
-[CAUTION]
-These are the only steps which aren't idempotent and have to be completed uninterrupted in one go.
-If you have to recreate the install config or any of the generated manifests you need to rerun all of the subsequent steps.
-
-[NOTE]
---
-You can add more options to the `install-config.yaml` file.
-Have a look at the https://docs.openshift.com/container-platform/{ocp-minor-version}/installing/installing_bare_metal/installing-bare-metal.html#installation-bare-metal-config-yaml_installing-bare-metal[config example] for more information.
-
-For example, you could change the SDN from a default value to something a customer requests due to some network requirements.
---
+include::partial$install/installer-notes.adoc[]
 
 . Prepare SSH key
 +
-NOTE: We generate a unique SSH key pair for the cluster, because Exoscale doesn't support configuring multiple public keys for a VM.
+[NOTE]
+====
+We generate a unique SSH key pair for the cluster, because Exoscale doesn't support configuring multiple public keys for a VM.
+====
```

Comment:
This change seems unnecessary, but I don't really care either way.

Reply:
I know, but it was making my VS code happier as it couldn't do syntax highlighting for the rest of the document :shrug: